torchtitan/datasets/hf_datasets.py

```diff
@@ -102,9 +112,20 @@ def __init__(
         else:
             ds = load_dataset(dataset_path, split="train")
 
+
+        if data_processing_style in _supported_data_processing_styles:
+            data_processing_fn = eval(_supported_data_processing_styles[data_processing_style])
```

Comment:
Why do we use eval here?

Reply:
maybe something like this:
```python
try:
    data_processing_fn = _supported_data_processing_styles[style]
except KeyError as e:
    raise ValueError(f"Unsupported data processing style: {style}") from e```
    
    Could even make this an enum which maps to a Callable type.

Reply:
Ok, good point, I'll update it.